server/db/init.js

```diff
@@ -22,7 +22,53 @@ bookshelf.knex.schema.hasTable('users').then((exist) => {
       table.timestamp('date_created').defaultTo(bookshelf.knex.fn.now());
       table.timestamp('date_updated').defaultTo(bookshelf.knex.fn.now());
     }).then(() => {
-      console.log(('Created users table'));
+      console.log('Created users table');
     });
   }
 });
+
+bookshelf.knex.schema.hasTable('scripts').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('scripts', (table) => {
+      table.increments('id').primary();
+      table.string('name').notNullable().index();
```

Comment:
Yeah, we still need to speak to the team about which fields we want to require unique values, I don't think we got to that yesterday.